Doc/library/os.rst

```diff
@@ -569,6 +569,28 @@ process and user.
       See the documentation for :func:`getgroups` for cases where it may not
       return the same group list set by calling setgroups().
 
+.. function:: setns(fd, nstype=0)
+
+   Reassociate thread with a namespace.
+   If *fd* refers to a ``/proc/[pid]/ns/`` link, :func:`setns` reassociates the
```

Comment:
There is no need to link back to the function we are describing :) You can replace the `:func:setns` links with ` ``setns()`` ` in this paragraph.